gearcmd/worker.go

```diff
@@ -40,11 +40,17 @@ func (conf TaskConfig) Process(job baseworker.Job) ([]byte, error) {
 	}
 	if err != nil {
 		data["error_message"] = err.Error()
-		log.Printf(kayvee.FormatLog("gearcmd", "error", "END_WITH_ERROR", data))
+		data["type"] = "gauge"
+		data["value"] = 0
+		data["success"] = false
```

Comment:
That requires configuring the `.toml` file and setting the `job_id`, `job_data`, `success` as part of the ignored fields. I am using only `source`, `function` and `title` in accepted fields (PR for that coming up).